docs/agile-process.md

```diff
@@ -62,9 +66,14 @@ Time-based estimations are prone to inaccuracy and preconceived notions of "how
 
 # Supporting Meetings and processes
 
+### Standup
+We do standups on video chat, but everyone has the option to do it in Slack instead. 
+
 ### Sprint planning (Backlog->Available)
 
-At the start of each sprint, we review the stories refined for implementation and ensure they . We aim to take on work that can be completed within one sprint. If a story seems like it'll need to span more than one sprint, that may be a sign that it needs to be broken into a smaller chunk of work.
+At the start of each sprint, we review the stories refined for implementation and ensure they . We aim to take on work that can be completed within one sprint. If a story seems like it'll need to span more than one sprint, that may be a sign that it needs to be broken into a smaller chunk of work. 
```

Comment:
> ... we review the stories refined for implementation and ensure they . We aim to take on work ...

Looks like this is missing some words.

Reply:
Ah, yep! Updated.

Reply:
I think I'm still seeing the old version 😕 Don't know if it's just the GH interface being weird

Reply:
Heh, yeah I think it's the GH interface being a little weird. Seeing it here: https://github.com/GSA-TTS/FAC/pull/356/files#diff-b8b5a8355703172e0eae03521915c15266ad6d3c2307818a59cbec03f4d26b86R74

Btw, I'm open to having this doc live in google docs if that's preferable.